src/ray/gcs/gcs_server/gcs_server.cc

```diff
@@ -181,6 +185,11 @@ void GcsServer::StoreGcsServerAddressInRedis() {
   RAY_LOG(INFO) << "Finished setting gcs server address: " << address;
 }
 
+bool GcsServer::Ping(const std::string &ip, int port) {
+  AsyncClient client;
+  return client.Connect(ip, port, 1000);
```

Comment:
Could you replace this hardcoded `1000 ms` with `RayConfig::instance().internal_gcs_service_connect_wait_milliseconds()`

Reply:
fixed, thanks